main.js

```diff
@@ -0,0 +1,21 @@
+const listedLinks = document.querySelectorAll('li')
+for (let listedLink of listedLinks) {
+    listedLinks[0].remove()
+}
+
+const reads = document.querySelectorAll('span')
+for (let read of reads) {
```

Comment:
The instructions specified 1 span to remove but your code has removed all span elements